addons/web_editor/static/src/xml/image_link_tools.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="web_editor.image_link_tools_button">
+        <we-button class="fa fa-link" data-set-link="" data-no-preview="true"
+            title="Redirect the user elsewhere when he clicks on the image."/>
+    </t>
+
+    <t t-name="web.editor.image_link_tools_fields">
+        <t t-set="url_input_label">⌙ Your URL</t>
+        <we-urlpicker t-att-string="url_input_label"
+            data-set-url="" name="url" data-name="link_on_image_opt"
+            title="Type '/' to search a page. '#' to link to an anchor."
```

Comment:
I guess you use the exact same term as the existing one here, but I am not sure if we have something to do for this to use the correct translation. Could you try and check if that works?